stdlib/universe/universe.flux

```diff
@@ -381,3 +381,6 @@ toUInt = (tables=<-) => tables |> map(fn: (r) => ({r with _value: uint(v: r._val
 toFloat = (tables=<-) => tables |> map(fn: (r) => ({r with _value: float(v: r._value)}))
 toBool = (tables=<-) => tables |> map(fn: (r) => ({r with _value: bool(v: r._value)}))
 toTime = (tables=<-) => tables |> map(fn: (r) => ({r with _value: time(v: r._value)}))
+
+// today returns the now() timestamp truncated to the day unit
+today = () => date.truncate(t: now(), unit: 1d)
```

Comment:
I like this function but would like to see it as part of the `date` package instead of the universe.

Reply:
Hmm ok lets add it to universe then, thanks!